release.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -euxo pipefail
+
+rm -rf release || exit 0;
+
+elm-package bump
+
+cleanup="examples gh-pages.sh pipeline.png CONTRIBUTING.md .eslintrc.json release.sh"
+version=$(grep -m1 version elm-package.json | awk -F: '{ print $2 }' | sed 's/[", ]//g')
+
+git commit -a -m "Bump to $version"
+git push
+
+last_commit=$(git rev-parse HEAD)
+
+git clone --reference . git@github.com:elm-community/webgl.git release
```

Comment:
Oh, I see what you mean. I really want a clean folder, to not contain some leftovers, this also allows to `cd` into it and review things that were pushed.